napari/utils/colormaps/vendored/cm.py

```diff
@@ -23,9 +23,9 @@
 import numpy as np
 from numpy import ma
 
-from napari.utils.colormaps.vendored import colors
-from napari.utils.colormaps.vendored._cm import datad
-from napari.utils.colormaps.vendored._cm_listed import cmaps as cmaps_listed
+from . import colors
+from ._cm import datad
+from ._cm_listed import cmaps as cmaps_listed
```

Comment:
This is in a module called `vendored`, but is actually original napari code, right?

Reply:
I do not know.

Reply:
These were added in #89. @jni, should we treat this module (or part of it) as vendored the same way as the `_vendor` module?

Reply:
I'm sorry that was, erm, some time ago 😅 and I don't remember all the changes that might have been added since that PR (or even between matplotlib and that PR). I am 100% sure that we were not as careful then about "isolating" vendored code — we just wanted the mpl colormaps without depending on mpl, so I copied the necessary files over.

If we want to do things "right", we need to determine the commit history (did it get reformatted with black, for example?), whether we can update with some newer mpl maps, have a fallback where if mpl is installed we use the modules from mpl but if not we use these, etc.

Reply:
I think for now then it doesn't really matter; let's not bother with it in this PR and open a separate issue/pr about it. Sound good?

Reply:
100%